src/test/java/api/queue/RequestQueueResourceTest.java

```diff
@@ -612,8 +612,9 @@ private IndividualResource pageTitleLevelRequestForItem(IndividualResource reque
       .titleRequestLevel()
       .open()
       .page()
+      .withNoItemId()
+      .withNoHoldingsRecordId()
       .withInstanceId(item.getInstanceId())
-      .forItem(item)
       .by(requester)
       .withPickupServicePoint(servicePointsFixture.cd1()));
```

Comment:
Do we need this method? pageTitleLevelRequestForItem(IndividualResource requester,
    ItemResource item)

Reply:
This method is needed to improve readability.